### PR TITLE
Inser data into animals table and queried the data

### PR DESCRIPTION
**Pull request by osbentech**
This is a Postgres project named vet_clinic.
Here I created data and queried them.
below are the screenshots of my terminal:
image
image
image
image
image
image
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -9,4 +9,49 @@ SELECT date_of_birth FROM animals WHERE name IN('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT name, neutered FROM animals WHERE neutered = True;
 SELECT name FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+BEGIN;
```
Comment (BertrandConxy) [inline queries.sql:15]: - [ ] In the project requirements, there is a point about the queries you have not accounted for. Take a look at the following screenshot 👇 
image

Please add the query as mentioned above here also 😃 👍

File: queries.sql [hunk 2]
```diff
@@ -9,4 +9,49 @@ SELECT date_of_birth FROM animals WHERE name IN('Agumon', 'Pikachu');
...
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+ROLLBACK SP1;
+UPDATE animals
+SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+-- aggregate
+
+SELECT COUNT(name) FROM animals;
```
Comment (BertrandConxy) [inline queries.sql:42]: - [ ] After each transaction, please take a screenshot and then add them to the PR summary for further verification 😃 👍

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @osbentech, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- correct flow is used

## Required Changes ♻️

- [ ] Good job so far 👏 your Pull request description/ summary needs improvement. Please describe briefly what you have done in this current branch. On top of that, It is among the project requirements to add the screenshot of query results in your PR summary 😃 👍  So please add them 😃 

- [ ] You have not added the command to add the `species` column in the `animals` table. Please add that command in the `schema.sql` to meet the project requirements 😃 👍 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Benjamin, 
## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**